src/App.res

```diff
@@ -13,6 +13,20 @@ let make = () => {
   let onCreateTarget = _evt => dispatch(SetDialog(CreateTarget))
   let onOpenHelp = _evt => dispatch(SetDialog(Help))
 
+  let getnumberOfUntranslatedSegments = (data: Source.t) => {
+    data
+    ->Array.filter(cell =>
+      cell
+      ->Array.get(3)
+      ->Option.map(target => target.value->String.length === 0)
+      ->Option.getOr(false)
```

Comment:
`cell[3]->Option.mapOr(false, target => target.value->String.length === 0)`
is a bit shorter

Reply:
Done!